### PR TITLE
HUE-9513 [core] Improve Hue SAML response check: add defensive code i…

### PR DESCRIPTION
**Pull request by ranade1**
…nvalid SAML case

(cherry picked from commit 9a6c86bbd99833fb2976d67763c15b1711f6ceed)

### CODE REVIEW

File: desktop/core/src/desktop/views.py [hunk 1]
```diff
@@ -92,13 +92,19 @@ def samlgroup_check(request):
...
 
-      if json_data.get('saml_attributes', False) and \
-         json_data['saml_attributes'].get(REQUIRED_GROUPS_ATTRIBUTE.get(), False):
-        success = set(REQUIRED_GROUPS.get()).issubset(
-                 set(json_data['saml_attributes'].get(REQUIRED_GROUPS_ATTRIBUTE.get())))
-        if not success:
-          LOG.info("User %s not found in required SAML groups, %s" % (request.user.username, REQUIRED_GROUPS.get()))
-          return False
+      if json_data.get('saml_attributes', False):
+        LOG.info("Empty saml_attributes data for %s user" % (request.user.username))
```
Comment (romainr) [inline desktop/core/src/desktop/views.py:96]: (request.user.username) -- > request.user.username

**Review by romainr — APPROVED** [review]

File: desktop/core/src/desktop/views.py (hunk 1 above)
Comment (ranade1) [reply desktop/core/src/desktop/views.py:96]: it is a valid python way.

**Comment by ranade1** [comment]
recheck

**Comment by ranade1** [comment]
recheck

**Comment by romainr** [comment]
Rebase on master and added pylint fix in https://github.com/cloudera/hue/pull/1342

https://github.com/cloudera/hue/commit/f853ea0e8c591efcaa55ae69b3c07ee0ca5d7dda
https://github.com/cloudera/hue/commit/6c6904fe3ce47c629993569d3ba1c06b3bb8a5ed

(so the same code from here is in master)

**Pull request closed without merging**